heron_wsgi/admin_lib/i2b2pm.py

```diff
@@ -293,21 +306,26 @@ def authz(self, uid, full_name,
         else:
             log.info('found: %s', me)
             me.password, me.status_cd, me.change_date = pw_hash, 'A', t
-        # http://docs.sqlalchemy.org/en/rel_0_8/orm/query.html?highlight=query.update#sqlalchemy.orm.query.Query.update # noqa
-        ds.query(UserRole).filter(and_(
-            UserRole.user_id == uid,
-            UserRole.user_role_cd.in_(list(roles)))).\
-            delete(synchronize_session='fetch')
-
-        # If a user has permissions to REDCap i2b2 project,
-        # also grant permissions to default project #2111
-        for project in set([project_id, DEFAULT_PID]):
-            for r in roles:
-                myrole = UserRole(user_id=uid, project_id=project,
-                                  user_role_cd=r,
-                                  entry_date=t, change_date=t, status_cd='A')
-                log.info('I2B2PM: adding: %s', myrole)
-                me.roles.append(myrole)
+
+        admin_role = ds.query(UserRole).filter_by(user_id=uid,
+                                                  user_role_cd='ADMIN').all()
+
+        if not admin_role:
```

Comment:
This won't let Maren (or anybody else with ADMIN) get access to REDCap projects (i.e. `project_id`), right?